dds_web/database/models.py

```diff
@@ -179,6 +180,26 @@ def __repr__(self):
         return f"<Email {self.email}>"
 
 
+class Invite(db.Model):
+    """Invites for users not yet confirmed in DDS"""
+
+    # Table setup
+    __tablename__ = "invites"
+    __table_args__ = {"extend_existing": True}
+
+    # Columns
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    email = db.Column(db.String(80), unique=True, nullable=False)
```

Comment:
Apparently 255 characters is the maximum for an email address: https://stackoverflow.com/a/1199238/1472158, might not be worth the trouble of having the string field smaller than that?